pymarl/runners/parallel_runner.py

```diff
@@ -3,12 +3,6 @@
 from pymarl.components.episode_buffer import EpisodeBatch
 from multiprocessing import Pipe, Process
 import numpy as np
-import torch as th
-from pymarl.modules.bandits.const_lr import Constant_Lr
-from pymarl.modules.bandits.uniform import Uniform
-from pymarl.modules.bandits.reinforce_hierarchial import EZ_agent as enza
-from pymarl.modules.bandits.returns_bandit import ReturnsBandit as RBandit
-import time
```

Comment:
Do we have to delete "time" as well here?

Reply:
It is unused.
I think it is there because I used to add some time.sleep() to check multiprocess behavior months ago.